echo-telemetry/echo-telemetry.gradle

```diff
@@ -23,4 +23,5 @@ dependencies {
   implementation 'com.netflix.spinnaker.kork:kork-web'
   implementation 'com.squareup.retrofit:retrofit'
   implementation 'com.squareup.retrofit:converter-jackson'
+  implementation 'io.github.resilience4j:resilience4j-spring-boot2'
```

Comment:
This shouldn't be necessary. I added it as an `api` scope to `kork-core`.

https://github.com/spinnaker/kork/blob/master/kork-core/kork-core.gradle#L18

Reply:
I think @ezimanyi was pushing for each module being more explicit about it's direct dependencies, no?

Reply:
I think if we add it explicitly as `api` scope in `kork-core` this is fine; I was more concerned with depending on things that were in `implementation` scope in transitive dependencies (that would nonetheless end up on the compile classpath in gradle 4).

Reply:
Removed because of the transitive dep of echo-telemetry --> echo-model --> kork-core --> resilience4j-spring-boot2